upi/vsphere/terraform.tfvars.example

```diff
@@ -75,3 +78,20 @@ ipam_token = "TOKEN_FOR_THE_IPAM_SERVER"
 // The IP addresses to assign to the compute VMs. The length of this list must
 // match the value of compute_count.
 //compute_ips = ["10.0.0.30", "10.0.0.31", "10.0.0.32"]
+
+// NSX part for NSX-T network.
```

Comment:
While many deployments will have NSX-T not all deployments will be on NSX-T. We need a variable to switch NSX-T on/off. If off, the NSX-T resources won't be created.